src/commands/send.rs

```diff
@@ -169,6 +175,126 @@ fn print_response(blob: Vec<u8>, method_name: String) -> AnyhowResult {
             let response = Decode!(blob.as_slice(), ManageNeuronResponse)?;
             println!("Response: {:?\n}", response);
         }
+        SupportedResponse::SnsCanistersSummary => {
+            let response: Vec<(String, PrincipalId, CanisterStatusResultV2)> = Decode!(
+                blob.as_slice(),
+                Vec<(String, PrincipalId, CanisterStatusResultV2)>
+            )?;
+            for (name, principal, canister_status) in response.into_iter() {
+                println!("\nCanister Summary:");
+                println!("name: {}", name);
+                println!("id: {}", principal);
+                println!("controller: {}", canister_status.controller());
+                println!("status: {}", canister_status.status());
+                println!(
+                    "module hash: {}",
+                    &canister_status
+                        .module_hash()
+                        .map_or("None".to_string(), |hash| hex::encode(hash))
+                );
+                println!("memory size: {}", canister_status.memory_size());
+                println!("cycles: {}", canister_status.cycles());
+                println!(
+                    "freezing threshold: {}",
+                    canister_status.freezing_threshold()
+                );
+            }
+        }
+        SupportedResponse::NervousSystemParameters => {
+            let response = Decode!(blob.as_slice(), NervousSystemParameters)?;
+            println!("\nNervous System Parameters:");
+            println!(
+                "reject cost e8: {}",
```

Comment:
I think we should either go with the underscores in the name, or capitalize the words. (I vote underscores as they are parameters)